development/image-syncer/external-images.yaml

```diff
@@ -43,6 +43,7 @@ images:
 - source: "quay.io/prometheus/prometheus:v2.32.1"
 - source: "quay.io/service-manager/sb-proxy-k8s:v0.9.1"
 - source: "nats@sha256:ae7411548aaccfbe8c77b0f055c26ca31e0f446b32d5e0ac2057d071c679adab"
```

Comment:
this one is already synced, i guess we can delete it

Reply:
done

Reply:
we cannot change the SHA attached to a tag which is already synced so I added a new tag.